todoman/cli.py

```diff
@@ -52,13 +52,13 @@ def _todo_property_options(command):
         help=('The due date of the task, in the format specified in the '
               'configuration file.'))(command)
     click.option(
-        '--start', '-s', default='', callback=_validate_date_param,
+        '--dtstart', '-s', default='', callback=_validate_date_param,
```

Comment:
I thought similarly, but dtstart (the Todo property) is exposed through the `--sort` option, so we would have to rename that as well.